graphrag/index/operations/build_noun_graph/build_noun_graph.py

```diff
@@ -122,7 +123,7 @@ def _extract_edges(
         strict=False,
     )
```

Comment:
This code can be replaced by this, to remove the applies used (also tested locally on my branch)

```python
edge_df[["source", "target"]] = edge_df["edges"].to_list()
edge_df["min_source"] = edge_df[["source", "target"]].min(axis=1)
edge_df["max_target"] = edge_df[["source", "target"]].max(axis=1)
edge_df = edge_df.drop(columns=["source", "target"]).rename(columns={"min_source": "source", "max_target": "target"})

edge_df = edge_df[(edge_df.source.notna()) & (edge_df.target.notna())]
edge_df = edge_df.drop(columns=["edges"])
```